samples/drivers/comparator/pending/README.rst

```diff
@@ -0,0 +1,32 @@
+.. zephyr:code-sample:: comparator_pending
+   :name: Comparator pending
+   :relevant-api: comparator_interface
+
+   Monitor the output of a comparator
```

Comment:
Both samples have the same description?

Reply:
I think I may replace the samples with an "empty" sample so users can just use the shell to test with. WDYT?

Reply:
I'm typically not fond of "empty" samples just for showcasing a shell module.

How about adding shell documentation instead (see e.g. https://docs.zephyrproject.org/latest/hardware/peripherals/can/shell.html and https://docs.zephyrproject.org/latest/hardware/peripherals/eeprom/shell.html) with details on how to enable the shell module? Along with a proper test of the shell module (see e.g. https://github.com/zephyrproject-rtos/zephyr/tree/main/tests/drivers/can/shell)?

Reply:
@henrikbrixandersen sounds good, I will add shell docs + tests and remove the sample :)